go/libraries/doltcore/sqle/dsess/session.go

```diff
@@ -757,55 +757,60 @@ func (d *DoltSession) GetRoots(ctx *sql.Context, dbName string) (doltdb.Roots, b
 // ResolveRootForRef returns the root value for the ref given, which refers to either a commit spec or is one of the
 // special identifiers |WORKING| or |STAGED|
 // Returns the root value associated with the identifier given and its commit time
-func (d *DoltSession) ResolveRootForRef(ctx *sql.Context, dbName, hashStr string) (*doltdb.RootValue, *types.Timestamp, error) {
+func (d *DoltSession) ResolveRootForRef(ctx *sql.Context, dbName, hashStr string) (*doltdb.RootValue, *types.Timestamp, string, error) {
```

Comment:
Is the returned hash string value always going to be the same as the `hashStr` input param? From reading this code, it seems like `hashStr` always needs to be a hash string (or the special `WORKING` or `STAGED` values). I'm guessing that this is actually being used to take in branch names or tags, too? 

If that's the case... then I think it's worth updating the `hashStr` variable name to `ref` or something so that the code is more self-describing (otherwise a reader would assume that `hashStr` has to always be a hash string).